src/ECharts/Event.purs

```diff
@@ -37,3 +39,43 @@ listenAll chart cb =
 
   toEChartsEvent ∷ ∀ ω. Tuple String ω → EChartsEvent
   toEChartsEvent = unsafeCoerce
+
+foreign import dispatchAction_
+  ∷ ∀ e action
+  . action

```

Comment:
can we change action to be `{ type ∷ String | r }`?

Reply:
Discussed in Slack